### ISSUE TITLE
No more log file being written

### ISSUE THREAD

**Issue by def-** [ddnet/ddnet7#63]
I updated the servers 3 days ago. No more log files are written since then. even though "logfile servers/9303.log" is set.

**Comment by ChillerDragon** [comment]
Did you create the folder ``~./teeworlds/dumps/servers``?
New log system does not allow logfiles outside the dumps directory.

**Comment by def-** [comment]
Nope, and I also don't want logs to end up there. Also, the server should create the directory on its own, shouldn't it?

**Comment by ChillerDragon** [comment]
> Also, the server should create the directory on its own, shouldn't it?

It does not, im not sure if we should change that.

**Comment by def-** [comment]
Then you can also argue that the player has to create their own .teeworlds directory.
